Projects/App/Sources/UI/Place/AddPlace/AddCategoryViewController.swift

```diff
@@ -0,0 +1,156 @@
+//
+//  AddCategoryViewController.swift
+//  App
+//
+//  Created by Chanhee Jeong on 2022/10/24.
+//  Copyright (c) 2022 zesty. All rights reserved.
+//
+
+import Combine
+import UIKit
+import DesignSystem
+import SnapKit
+import Kingfisher
+
+final class AddCategoryViewController: UIViewController {
+    
+    // MARK: - Properties
+    private let viewModel = AddPlaceViewModel()
+    private var cancelBag = Set<AnyCancellable>()
+    private var place: Place?
+    private var reviews: [Review] = []
+    private var categories: [Category] = Category.mockData
+    
+    private var selectedIcon: String = ""
+    
+    private lazy var titleView = MainTitleView(title: "선택한 맛집에서는\n무엇을 먹을 수 있나요?")
+    
+    private lazy var categoryCollectionView: UICollectionView = UICollectionView(frame: .zero, collectionViewLayout: createCollecionViewLayout())
+    
+    private lazy var addPlaceButton: UIButton = {
+        $0.configuration = .borderedTinted()
+        $0.setTitle("맛집 등록하기", for: .normal)
+        $0.addTarget(self, action: #selector(addPlaceButtonDidTap), for: .touchUpInside)
+        return $0
+    }(UIButton())
+    
+    // MARK: - LifeCycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setNavigationBar()
+        configureUI()
+        createLayout()
+    }
+    
+    // MARK: - Function
+    @objc func backButtonDidTap() {
+        self.navigationController?.popViewController(animated: true)
+    }
+    
+    @objc func addPlaceButtonDidTap() {
+        self.navigationController?.pushViewController(AddPlaceResultViewController(), animated: true)
+    }
+    
+}
+
+// MARK: - Binding
+extension AddCategoryViewController {
+    
+    private func bind() {
+    }
+
+}
+
+// MARK: - UI Function
+
+extension AddCategoryViewController {
+    
+    private func configureUI() {
+        view.backgroundColor = .white // zestyColor(.backgroundColor)
+        categoryCollectionView.register(CategoryIconCell.self, forCellWithReuseIdentifier: "CategoryIconCell")
+        categoryCollectionView.isScrollEnabled = false
+        categoryCollectionView.delegate = self
+        categoryCollectionView.dataSource = self
+    }
+    
+    private func createLayout() {
+        view.addSubviews([titleView, categoryCollectionView, addPlaceButton])
+        
+        titleView.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide)
+            $0.horizontalEdges.equalToSuperview().inset(20)
+        }
+        
+        categoryCollectionView.snp.makeConstraints {
+            $0.horizontalEdges.equalToSuperview().inset(20)
+            $0.top.equalTo(titleView.snp.bottom).offset(25)
+            $0.bottom.equalTo(view.safeAreaLayoutGuide).inset(90)
+        }
```

Comment:
p1;

categoryCollectionView의 하단을 safeArea를 기준으로 잡기보다는 addPlaceButton의 상단을 기준으로 간격을 줘야할 것 같습니다. 지금 상태는 categoryCollectionView와 addPlaceButton이 겹칠 여지가 있어서 고쳐야할것 같아요!

```suggestion
        categoryCollectionView.snp.makeConstraints {
            $0.horizontalEdges.equalToSuperview().inset(20)
            $0.top.equalTo(titleView.snp.bottom).offset(20)
            $0.bottom.equalTo(addPlaceButton.snp.top).offset(-20)
        }
```

이왕 바꾸는거 상하 간격을 동일하게 주는게 어떨까 해서 top offset도 20으로 바꿔봤습니다😝

Reply:
네